### PR TITLE
feat(playwright): add `testCaseId` fix `fullName` and `historyId`

### PR DESCRIPTION
**Pull request by vovsemenv**

### CODE REVIEW

File: packages/allure-playwright/src/index.ts [hunk 1]
```diff
@@ -84,8 +87,20 @@ class AllureReporter implements Reporter {
     if (suiteTitles.length > 0) {
       allureTest.addLabel(LabelName.SUB_SUITE, suiteTitles.join(" > "));
     }
-    allureTest.historyId = test.titlePath().slice(1).join(" ");
-    allureTest.fullName = test.title;
+    const project = suite.project()!;
+    if (project?.name) {
+      allureTest.addParameter("project", project.name, { hidden: true });
```
Comment (baev) [inline packages/allure-playwright/src/index.ts:92]: why hidden?

File: packages/allure-playwright/src/index.ts [hunk 2]
```diff
@@ -203,6 +218,14 @@ class AllureReporter implements Reporter {
         runtime.writeAttachment(stderr, "text/plain"),
       );
     }
+
+    allureTest.historyId = md5(
```
Comment (baev) [inline packages/allure-playwright/src/index.ts:222]: is not required, it will be calculated this way on the report/testops site

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/allure-playwright/src/index.ts (hunk 2 above)
Comment (vovsemenv) [reply packages/allure-playwright/src/index.ts:222]: Is it ok i if i completely remove historyId from playwright integration? previously we used it [there](https://github.com/allure-framework/allure-js/blob/a1a1132128a346ac8453e206b287f63931e999bc/packages/allure-playwright/src/index.ts#L87)

File: packages/allure-playwright/src/index.ts [hunk 3]
```diff
@@ -33,6 +34,8 @@ import {
 } from "allure-js-commons";
 import { ALLURE_METADATA_CONTENT_TYPE } from "allure-js-commons/internal";
 
+const md5 = (val: string) => createHash("md5").update(val).digest("hex");
```
Comment (baev) [inline packages/allure-playwright/src/index.ts:37]: maybe move it to commons ?

**Review by baev — APPROVED** [review]

**Comment by vovsemenv** [comment]
@baev ready

**Comment by vovsemenv** [comment]
@baev merge pls

File: packages/allure-js-commons/src/ExecutableItemWrapper.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ import {
 export class ExecutableItemWrapper {
   constructor(private readonly info: FixtureResult | TestResult) {}
 
-  protected get wrappedItem(): FixtureResult | TestResult {
```
Comment (baev) [inline packages/allure-js-commons/src/ExecutableItemWrapper.ts:19]: why change?

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/allure-js-commons/src/ExecutableItemWrapper.ts (hunk 4 above)
Comment (vovsemenv) [reply packages/allure-js-commons/src/ExecutableItemWrapper.ts:19]: it was necessary when i had computed `historyId`. cleaned up

**Review by baev — APPROVED** [review]

**Comment by baev** [comment]
@vovsemenv thanks!

**Pull request merged**